pkg/services/team/create.go

```diff
@@ -2,10 +2,14 @@ package team
 
 import (
 	"context"
+
 	"github.com/jairogloz/go-l/pkg/domain"
 )
 
 func (s Service) Create(ctx context.Context, team *domain.Team) (err error) {
```

Comment:
Es buen momento para empezar con la buena práctica de agregar comentarios a las funciones. La buena práctica es que los comentarios comiencen con el nombre exacto de la función y luego una descripción:

// Create creates a team in our system.

Reply:
Claro Jairo, lo comento y la próxima vez hago una branch desde mi branch verdad? Y luego esa feature/branch (nacida de mi branch/develop) le haga pr a develop?